### ISSUE TITLE
Unable to type in Japanese in Ubuntu GNOME

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#3013]
Though I am able to compose message in qTox in English and Russian, I have found that I am unable to compose them in Japanese (specifically I am using the "Japanese (Anthy)" input source) and I end up just typing in English though it works fine in other applications.

I am running Ubuntu GNOME 15.10 with GNOME 3.18 and I have got qTox version `1.3.0-32-g9d8561a-1` installed.

**Comment by ProMcTagonist** [comment]
Does it work in other Qt applications? What version of Qt do you have installed?

**Comment by ghost** [comment]
@ProMcTagonist: It does, and as far as I can tell I have Qt version 5 installed but I'm not sure of the actual package name so unless you can tell me that or how to find the more detailed version number that's all I can give you.

**Comment by zetok** [comment]
Related / duplicate of #2326 ?

**Comment by ghost** [comment]
Can you check if pasting Japanese characters work? #2564 

> how to find the more detailed version number

qTox -> Settings -> About

**Comment by ghost** [comment]
@kehugter: Ok, I have Qt version `5.4.2`, and yes, I am able to paste and send Japanese characters and my contacts are able to see them.

**Comment by ghost** [comment]
@zetok dupe?

**Comment by ghost** [comment]
@zetok, This issue is still present when using IBus though.

**Comment by ghost** [comment]
Unless I am not understanding something as I am not very knowledgeable in the area of input methods...

**Comment by zetok** [comment]
> @zetok, This issue is still present when using IBus though.

It could be an IBus issue?

Can you check if it works with other Qt5 apps?

**Comment by ghost** [comment]
@zetok, Well, I have tested it by setting the input method to "None", to "fcitx", and to "IBus", so it seems to be with all of them...

And as I think I mentioned before, this does not occur with other Qt5 apps, or any Qt apps at all, only qTox.

**Comment by zetok** [comment]
> And as I think I mentioned before, this does not occur with other Qt5 apps, or any Qt apps at all, only qTox.

Well, it would be good to know in which Qt5 apps it works – and it doesn't really matter whether it works with Qt4 apps.

> Ubuntu GNOME 15.10 with GNOME 3.18

I have doubts whether 15.10 ships with Qt5 apps, and this is why I'd like to actually know what those apps are.

**Comment by ghost** [comment]
@zetok, The file manager dolphin was one of them, if you want, I can get a list of the others?

**Comment by sudden6** [comment]
@SeventhSonOfASeventhSon Can you check if dolphin is a qt5 app? You can see the qt version of dolphin in help -> about dolphin or something similar.

**Comment by ghost** [comment]
@sudden6, I'm sure it is, I checked in its list of dependencies and lost of libs to do with Qt5 are there.

**Comment by ghost** [comment]
Dolphin's about info is useful, it tells which Qt version it is using and against which one it was built. You should post that.

**Comment by ghost** [comment]
@kehugter, Ok, well, it says it is using and was built against Qt version 5.5.1.

**Comment by Doom032** [comment]
Are you using Ubuntu 14.04 LTS? Because I'm also struck by #2326 on my home computer which uses this version of Ubuntu, but I found I don't have the problem on Debian Testing on my laptop. If all else fails, the next LTS version of Ubuntu, 16.04, will come out this april, and I think the updated software will probably have fixed this. If you can't wait that long but are still willing to update your OS, Ubuntu 15.10 is already out but will only be supported until july. #2564 is also relevant.

**Comment by ghost** [comment]
@Doom032, I already stated this information in my main post at the top.

**Comment by Doom032** [comment]
Oh, sorry. I was being stupid. And I was all hopeful too because I hate this bug.

**Comment by ghost** [comment]
A friend of mine tested this out using the normal Ubuntu 16.04 Beta 2 and apparently it worked as it should, though he did not get given the "Japanese (Anthy)" input source, just the "Japanese" one so I don't know if that could have been the difference, but anyway,he said it worked as expected... In other words this seems only to be a GNOME issue. Or an Ubuntu GNOME one, haven't tested this out yet on any other GNOME distros...

**Comment by SeventhSonOfASeventhSon** [comment]
My mistake, he was using the normal Ubuntu, but version 16.04 Beta 2 not 15.10.

**Comment by ghost** [comment]
Could your friend test if he gets the same results as I did in https://github.com/tux3/qTox/issues/2564#issuecomment-197530292?

**Comment by ghost** [comment]
@kehugter, Sorry, my friend had some issues with uninstalling the Japanese and is now reluctant to do any more testing work for this... I have installed Ubuntu 16.04 in a VM now but I can't seem to get Japanese to work anywhere so I'm not sure if I'm going to be much help... I will try reloading the VM and let you know if I can answer your questions or find anyone else without the issue who would be willing to do some testing.

**Comment by SeventhSonOfASeventhSon** [comment]
I mean I have installed Ubuntu 16.04, not Ubuntu GNOME because they only got it working in Ubuntu (Unity)...

**Comment by ghost** [comment]
@kehugter, Good news! I just installed Ubuntu 16.04 Beta 2 64-bit in a VM and Japanese works for me in qtox! So I will test to see if I get the same results as you do in the linked comment and get back to you...

**Comment by ghost** [comment]
@kehugter, Right, please see my comment here https://github.com/tux3/qTox/issues/2564#issuecomment-203658784.

**Comment by ghost** [comment]
I am now running Ubuntu GNOME 16.04 with GNOME 3.20 and the "Japanese (Anthy)" input source works, except for one small problem, when I type the `Type your message here...` text does not disappear and stays underneath. Is there a report already filed on that (I'm sure I've seen something around)?

**Comment by ghost** [comment]
Never mind, I found it: https://github.com/tux3/qTox/issues/3230 Shall this report be closed now?
